internal/buildscripts/packaging/fpm/etc/otel/collector/splunk-support-bundle.ps1

```diff
@@ -0,0 +1,260 @@
+# Copyright Splunk Inc.
```

Comment:
@djani-splunk Let's move this file to `internal/buildscripts/packaging/msi`.  Also we'll need to update https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/msi-builder/build.sh to include this file in the MSI.

Reply:
@jcheng-splunk I have added bundle script in MSI package build, and also I have added validation of bundle script in `windows-msi-validation` pipeline so for testing of the script no need to reinstall MSI separately, and validation can be done along with `windows-msi-validation`.
Please verify it and guide me If further changes are required on this PR.